unify.py

```diff
@@ -22,10 +22,12 @@
 
 from pathlib import Path
 
+
 class Header:
-  def __init__(self, path):
-    self.path = path
-    self.fulltext = path.read_text()
+  def __init__(self, include_path, open_path):
+    self.path = include_path
+    self.open_path = open_path
```

Comment:
This is never referred to again. Drop the assignment?

Reply:
done